### DISCUSSION TITLE
предложения / улучшения :: готика 2 Классик

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2_classic#100] [Ideas]
топик с багами https://github.com/dosinabox/g2_classic/discussions/88

**Comment by Kor-Angar** [comment]
мне кажется лучше сделать какую то запись в дневник при сдаче яиц беннету. сейчас не очевидно что квест закрыт.

```D
	else
	{
		AI_Output(self,other,"DIA_Bennet_EierBringen_06_06");	//Я не думаю, что ты найдешь еще яйца. К тому же, мне и этих достаточно. Я даже не знаю, что я буду делать со всеми ними.
		TOPIC_END_DRACHENEIER = TRUE;
	};
```

**Reply by Kor-Angar** [reply]
![20221107173357_1](https://user-images.githubusercontent.com/49514764/200345552-1d8cf166-4378-4554-ac9d-05a157b1cee9.jpg)

**Reply by Kor-Angar** [reply]
@dosinabox  к тому же реплику можно не успеть прочитать или проскипать :-) а в неоф. обновление у тебя это как реализовано?

**Reply by dosinabox** [reply]
Так же. Просто закрытие квеста.

**Reply by dosinabox** [reply]
> так и остался весеть. в инфо функции не закрывается

В классике добавлю принудительное закрытие. В обновлении опыт начисляется в конце диалога, поэтому дневник проверяется и квест закрывается сам.

**Reply by Kor-Angar** [reply]
гуд

**Comment by Kor-Angar** [comment]
Ларса на острове можно одеть в доспехи наёмника

![20221107235945_1](https://user-images.githubusercontent.com/49514764/200414360-2d7bb242-fe75-471b-86f1-1405ad76a3c8.jpg)
